apps/frontend/src/lib/store/block.store.ts

```
@@ -0,0 +1,102 @@
+  noBlocks?: true
+}
+
+interface BlockState {
+  blocks: Block[]
+  blockId: string | null
+  isLoading: boolean
+  error: string | null
+  fetchBlocks: (
+    session: string,
+    spaceId: string
+  ) => Promise<FetchBlocksResult | void>
```

Biome

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)